bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTConfig.kt

```diff
@@ -39,6 +46,8 @@ data class BKTConfig internal constructor(
     private var backgroundPollingInterval: Long = DEFAULT_BACKGROUND_POLLING_INTERVAL_MILLIS
     private var appVersion: String? = null
     private var logger: BKTLogger? = DefaultLogger()
+    private var wrapperSdkVersion: String? = null
+    private var wrapperSdkSourceId: Int? = null
```

Comment:
Internally, we can treat it as `sdkVersion`. We just need to use the prefix `wrapper` in the public interface.
But if it's necessary, you can keep it as it is.

```suggestion
    private var sdkVersion: String? = null
    private var sdkSourceId: Int? = null
```